app/controllers/contacts_controller.rb

```diff
@@ -0,0 +1,68 @@
+class ContactsController < ApplicationController
+  before_action :find_contact, only: %i[edit update show destory]
+  before_action :find_multiple_contacts, only: %i[activate deactivate]
+
+  def index
+    @contacts = Contact.all
+  end
+
+  def show; end
+
+  def edit; end
```

Comment:
Can we provide a way to respond differently to htmx requests vs normal requests? Like if I wanted to go to `/contacts/1/edit` that should still work and it'd be useful to use this controller. I think @santib had something like
```suggestion
  def edit
    respond_to do |format|
      format.html { render partial: :form, locals: { contact: @contact } }
      format.html { render :edit }
    end
  end
```

Reply:
Sure! This is a feature we will add to the gem, but we did not in the current version yet and this examples are based on version `0.1.0` 🚀 🚀

Reply:
what about a helper in the ApplicationController? e.g.:
```diff
 class ApplicationController < ActionController::Base
+  def htmx_request?
+    request.headers["HX-Request"] == "true"
+  end
+
+  helper_method :htmx_request?
 end
```
this way the controller can render the template with or without the `layout`, deciding to send a complete HTML page or just a _fragment_:

```ruby
  def show
    @contact = fetch_contact(params[:id])
    render layout: !htmx_request?
  end
```